### DISCUSSION TITLE
How to convert this units?

### DISCUSSION THREAD

**Discussion by Achineseman** [Open-Systems-Pharmacology/Forum#943] [Questions & Problems]
<img width="418" alt="屏幕截图 2021-07-28 102553" src="https://user-images.githubusercontent.com/81560924/127262266-ec84ac58-cd47-402e-bcd9-53b9ba7d99b4.png">
<img width="960" alt="屏幕截图 2021-07-28 102536" src="https://user-images.githubusercontent.com/81560924/127262294-59451544-a4c2-4c0b-9b5e-9c81934969b7.png">

**Comment by prvmalik** [comment]
Clearance is in volume of reference fluid (mL) per unit time (min).
This clearance is accomplished per amount (pmol) of enzyme in a recombinant enzyme assay. To parameterize this you should have a rough idea of the amount in pmol of the enzyme (e.g. CYP3A4) that is in your assay. This can be difficult as recombinant assays tend to be quite variable even within labs.

Converting from your screen shot above, the uL/min/pmol enzymes for CYP2D6 can be converted directly.

The measurement per mg of Cyp enzymes likely refers to a hepatic microsomal assay which is handled by one of the other options in PK-Sim.

As an aside, it is extremely unlikely that these values will translate from in vitro to in silico/in vivo. Vmax values have very low translation potential. However KM values have great translation potential.
